File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -467,6 +467,170 @@ document.Close()
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Charts/Create-custom-chart).
 
+## Creating a Chart from Database
+
+Create a chart in a Word document using the .NET Word Library by utilizing the values retrieved from the database.
+
+The following code example illustrates how to create a chart in a Word document from database.
```

Comment:
The following code example illustrates how to create a chart in a Word document from a database.